testing/scenario_app/ios/Scenarios/ScenariosTests/AppLifecycleTests.m

```diff
@@ -122,7 +122,44 @@ - (void)testLifecycleChannel {
       addObject:[[XCTestExpectation alloc]
                     initWithDescription:
                         @"A second FlutterViewController goes through AppLifecycleState.resumed"]];
-  flutterVC = [rootVC showFlutter];
+  // At the end of second Flutter VC, we want to make sure it deallocs and sends detached signal.
+  // Using autoreleasepool will guarantee that.
+  FlutterViewController* flutterVC2;
+  @autoreleasepool {
```

Comment:
Sorry, unfortunately you'd have to rebase this one against master. I refactored this test a little bit.